src/combinators/index.spec.js

```diff
@@ -0,0 +1,19 @@
+const test = require('tape')
+
+const applyTo = require('./applyTo')
+const composeB = require('./composeB')
+const constant = require('./constant')
+const flip = require('./flip')
+const identity = require('./identity')
+const substitution = require('./substitution')
+
+test('combinators entry', t => {
+  t.equal(applyTo, applyTo, 'provides the T combinator (applyTo)')
```

Comment:
The should test the file exports against the index, so something like:
```js
const index = require('.')

test('combinators entry', t => {
  t.equal(index.applyTo, applyTo, 'provides the T combinator (applyTo)')
  ...
})
```
That way we are testing the availability of the functions on the object returned from a `require('crocks/combinators')`.

Should match this pattern in all the new specs.